.github/workflows/CI.yml

```
@@ -32,7 +32,7 @@ jobs:
       - uses: julia-actions/julia-buildpkg@v1
       - uses: julia-actions/julia-runtest@v1
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v4
         with:
```

See https://github.com/JuliaMath/SpecialFunctions.jl/pull/464#discussion_r1477866639:
```suggestion
        with:
          token: ${{ secrets.CODECOV_TOKEN }} # required
```